specification/src/main/asciidoc/platform/ServiceProviderInterface.adoc

```diff
@@ -1,64 +1,62 @@
 == Service Provider Interface
 
-The Java™ Platform, Enterprise Edition (Java
-EE) includes several technologies that are primarily intended to be used
-to extend the capabilities of the Java EE containers. In addition, some
-Java EE technologies include service provider interfaces along with
-their application programming interfaces. A Java EE profile may include
-some or all of these facilities, as described in
-link:#a3212[See Profiles]”.
+The Jakarta™ EE Platform includes several technologies that are primarily
+intended to be used to extend the capabilities of the Jakarta EE containers.
+In addition, some Jakarta EE technologies include service provider interfaces
+along with their application programming interfaces. A Jakarta EE profile may
+include some or all of these facilities, as described in link:#a3212[See Profiles]”.
 
-=== Java™ EE Connector Architecture
+=== Jakarta™ Connectors
 
 The Connector API defines how resource adapters
-are packaged and integrated with any Java EE product. Many types of
+are packaged and integrated with any Jakarta EE product. Many types of
 service providers can be provided using the Connector API and packaging,
-including JDBC drivers, JMS providers, and JAXR providers. All Java EE
-products must support the Connector APIs, as specified in the Connector
-specification.
+including JDBC drivers, Jakarta Messaging providers, and Jakarta XML Registries
+providers. All Jakarta EE products must support the Connector APIs, as specified
+in the Connector specification.
 
 The Connector specification is available at
 _http://jcp.org/en/jsr/detail?id=322_ .
 
-=== Java™ Authorization Service Provider Contract for Containers
+=== Jakarta™ Authorization
 
-The JACC specification defines the contract
-between a Java EE container and an authorization policy provider.
+The Jakarta Authorization specification defines the contract
+between a Jakarta EE container and an authorization policy provider.
 
-The JACC specification is available at
+The Jakarta Authorization specification is available at
 _http://jcp.org/en/jsr/detail?id=115_ .
 
-=== Java™ Transaction API
+=== Jakarta™ Transactions
 
-The Java Transaction API defines the
+The Jakarta Transactions defines the
 _TransactionSynchronizationRegistry_ interface that is intended for use
 by system level application server components such as persistence
-managers, resource adapters, as well as EJB and Web application
+managers, resource adapters, as well as Jakarta Enterprise Beans and Web application
 components. This provides the ability to register synchronization
 objects with special ordering semantics, associate resource objects with
 the current transaction, get the transaction context of the current
 transaction, get current transaction status, and mark the current
 transaction for rollback.
 
-The JTA specification is available at
+The Jakarta Transactions specification is available at
```

Comment:
Same comment here...  This link is the Java EE Transactions doc, not the Jakarta Spec...  We should have a single reference section at the end of the doc with references to the old and new Specs.  Maybe we should just be pointing at that?  Or, just remove this explicit reference.

Reply:
I agree. Can you point me to the new spec documents, please? I can't find them.

Reply:
We don't have the "standard" location for the updates specs yet...  At this point, let's just be consistent.  If we want to reference the JCP spec, then that's okay.  But, we shouldn't talk about Jakarta Spec and then point at the JCP location.